### ISSUE TITLE
Highlight errors and warnings in build output

### ISSUE THREAD

**Issue by kuvaldini** [microsoft/vscode-cmake-tools#1795]
### Expected:
Call "CMake: Build target"

OUPUT shold show ANSI xterm colors

### Apparent Behavior:
I see `[0;1;32m` instead of text in red color

### CMake Tools Log
```
[ 91%] Building CXX object irohad/ametsuchi/CMakeFiles/ametsuchi.dir/impl/peer_query_wsv.o
[build] [1m/Users/tanya/devel/iroha-review/irohad/maintenance/metrics.cpp:117:9: [0m[0;1;35mwarning: [0m[1munused variable '[domains_diff, peers_diff]' [-Wunused-variable][0m
[build]     auto[domains_diff,peers_diff] = calc_diffs(*pblock);
[build] [0;1;32m        ^
[build] [0m[1m/Users/tanya/devel/iroha-review/irohad/ametsuchi/impl/postgres_wsv_query.cpp:88:50: [0m[0;1;31merror: [0m[1minvalid operands to binary expression ('std::string' (aka 'basic_string<char, char_traits<char>, allocator<char> >') and 'std::string_view' (aka 'basic_string_view<char>'))[0m
[build]       auto msg = std::string{"Failed to count "} + what + ", query: " + e.what();
[build] [0;1;32m                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^ ~~~~
[build] [0m[1m/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/string:4100:1: [0m[0;1;30mnote: [0mcandidate template ignored: deduced conflicting types for parameter '_CharT' ('char' vs. 'std::__1::basic_string_view<char, std::__1::char_traits<char> >')[0m
[build] operator+(const basic_string<_CharT, _Traits, _Allocator>& __lhs, _CharT __rhs)
[build] [0;1;32m^
[build] [0m[1m/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/string:4163:1: [0m[0;1;30mnote: [0mcandidate template ignored: deduced conflicting types for parameter '_CharT' ('char' vs. 'std::__1::basic_string_view<char, std::__1::char_traits<char> >')[0m
[build] operator+(basic_string<_CharT, _Traits, _Allocator>&& __lhs, _CharT __rhs)
[build] [0;1;32m^
```

##### Looking at this I prefer to build manually in console :(

This is how it should look like 

<img width="1247" alt="изображение" src="https://user-images.githubusercontent.com/47349143/115609457-3893e780-a2f0-11eb-97ec-58d4d240b772.png">

**Comment by andreeis** [comment]
Thank you for opening this report. It is a duplicate of https://github.com/microsoft/vscode-cmake-tools/issues/1615.
The output channel window does not support colorization now and that feature is outside the scope of the CMake Tools extension (you can open a GitHub issue for the VSCode repository if you would like) but we still want to filter out those color codes to have the text readable.

**Comment by kuvaldini** [comment]
So, there is no way to get colored output?

**Comment by andreeis** [comment]
Not in the output channel window. Possibly in the terminal window, I didn't test to confirm.
Still, we fixed the problem with the code page codes being displayed and you can test it (if you want) with a vsix from here: https://github.com/microsoft/vscode-cmake-tools/actions/runs/811030625. This fix will be part of 1.7.2 which will be released later this week.
